### PR TITLE
add pyspelling

### PR DESCRIPTION
**Pull request by ocefpaf**
See https://facelessuser.github.io/pyspelling/

### CODE REVIEW

**Comment by goanpeca** [comment]
Hi @ocefpaf thanks for the recipe. It seems `hunspell` is not available on win ?

`conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform win-64: {'hunspell'}`

---

Also wondering, is there any suggestion on writing dependencies sorted alphabetically?

```yaml
run:
    - python >=3.6
    - beautifulsoup4
    - html5lib
    - lxml
    - markdown
    - pyyaml
    - soupsieve >=1.8
    - wcmatch >=6.0.3
    - hunspell
```

It does make reviewing a recipe easier, so I am curious about it :)

**Comment by ocefpaf** [comment]
> Hi @ocefpaf thanks for the recipe. It seems `hunspell` is not available on win ?

Since you are new in the team you probably don't know this but you only start reviewing a recipe when the submitter asked for a review. See https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-ping-team

Otherwise it is just noise while the submitter is testing things.
 
> Any other maintainers in mind that could help with the recipe?

Please do not block a recipe b/c of lack of maintainers! But do check if all the maintainers answered they want to be added. 

> Also wondering, is there any suggestion on writing dependencies sorted alphabetically?

Nope. I prefer the main dependency, like python, to be easily spotted. It doesn't really make any difference when reviewing, specially since most the recipes submitted by the conda-forge team are auto-generated.

**Comment by ocefpaf** [comment]
@conda-forge/staged-recipes this is ready for review!

File: recipes/pyspelling/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyspelling-{{ version }}.tar.gz
+  sha256: ba1d737d66c3f8992a183c5c7515bc9cb2ee5d2103310a969af32331bb70ec2a
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - pyspelling=pyspelling.__main__:main
+    - pyspelling3.9=pyspelling.__main__:main
```
Comment (xhochy) [inline recipes/pyspelling/meta.yaml:18]: Does it always provide a 3.9 suffixed entrypoint or is this dependent on the Python version?

Comment (ocefpaf) [reply recipes/pyspelling/meta.yaml:18]: I need to check the code. I'm pretty sure it is python dependant and, in that case, we'll need to patch it to make it truly noarch :-(

Comment (ocefpaf) [reply recipes/pyspelling/meta.yaml:18]: Oh fun! https://github.com/facelessuser/pyspelling/blob/3edf5857c972970a607df77ad5cc14fc01ab3949/setup.py#L48

Comment (ocefpaf) [reply recipes/pyspelling/meta.yaml]: @xhochy we remove versioned entrypoints in other places too. While I don't like to maintain patches, I believe this is the best I can do for now. What do you think?

Comment (xhochy) [reply recipes/pyspelling/meta.yaml]: Sounds fine.

**Pull request merged**